client/src/components/Nav Menu/NavMenu.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react';
+import { Link } from "react-router-dom";
+import './NavMenu.css'
+
+class NavMenu extends Component {
+
+    render() {
+        return (
+            <div className="nav-container">
+                <nav>
+                    <ul>
+                        <li>
+                                <h3>Home</h3>
+                            <span className="dots">&#9679;</span>
+                        </li>
+                        <li>
+                         
+                                <h3>Our Gowns</h3>
+                            <span className="dots">&#9679;</span>
+                        </li>
```

Comment:
I don't see where you're using the `Link` from React Router. Can you add that in, and for now just use a placeholder (`#`) for the `to` value